modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java

```diff
@@ -160,6 +166,25 @@ private GroupSearch _getGroupSearch(
 		return groupSearch;
 	}
 
+	private List<Group> _processGroups(Group group, List<Group> groups) {
+		if (!group.isStagingGroup()) {
+			return groups;
+		}
+
+		List<Group> digestedGroups = new ArrayList<>();
```

Comment:
I think Brian won't like this "digestedGroups" variable name, but I'll leave it to you.

Reply:
I miss this variable, I'll change it